dotnet/README.md

```diff
@@ -121,31 +121,29 @@ more features, such as connectors to services and Skills to perform specific act
 Unless you need to optimize which packages to include in your app, you will usually
 start by installing this meta-package first:
 
-* **Microsoft.SemanticKernel**
+* **[Microsoft.SemanticKernel](https://www.nuget.org/packages/Microsoft.SemanticKernel/)**
 
 This meta package includes core packages and OpenAI connectors, allowing to run
 most samples and build apps with OpenAI and Azure OpenAI.
 
 Packages included in **Microsoft.SemanticKernel**:
 
-1. **Microsoft.SemanticKernel.Abstractions**: contains common interfaces and classes
+1. [Microsoft.SemanticKernel.Abstractions](https://www.nuget.org/packages/Microsoft.SemanticKernel.Abstractions/): contains common interfaces and classes
```

Comment:
Can we make these bulleted lists instead of numbered? (there's no significance to the ordering)
- item 1
- item 2

Reply:
Sure I will get this done in next commit!